qutebrowser/mainwindow/prompt.py

```diff
@@ -709,10 +719,25 @@ def item_focus(self, which):
         if not idx.isValid():
             idx = last_index if which == 'prev' else first_index
 
+        idx = self._do_completion(idx, which)
+
         selmodel.setCurrentIndex(
             idx, QItemSelectionModel.ClearAndSelect | QItemSelectionModel.Rows)
         self._insert_path(idx, clicked=False)
 
+    def _do_completion(self, idx, which):
+        filename = self._file_model.fileName(idx)
+        while (not filename.startswith(self._to_complete)) \
```

Comment:
You don't need a `\` inside parentheses.

Reply:
The parentheses are just for the not - to make precedence more clear. If you don't like that I can get rid of it!